### DISCUSSION TITLE
Nominate Michael Dawson as a builder maintainer

### DISCUSSION THREAD

**Discussion by ForestEckhardt** [paketo-buildpacks/feedback#244] [Builder Team]
I'd like to nominate Michael Dawson (@mhdawson) as a builders maintainer. Michael implemented the UBI builder in the Paketo Community see https://github.com/paketo-community/builder-ubi-base and https://github.com/paketo-community/builder-ubi-buildpackless-base. These have given Paketo users access to a workflow using supported UBI images.

Now that UBI is moving in a release stage it makes sense to allow Michael the ability to upgrade and update the UBI builders using the subject matter expertise that they have without having to wait for another builder maintainer to get to the PR in question. 

@paketo-buildpacks/builders-maintainers can I get a 👍 / 👎 ?

**Comment by mhdawson** [comment]
@ForestEckhardt wondering what the next step is ?

**Comment by ForestEckhardt** [comment]
@paketo-buildpacks/steering-committee @robdimsdale is currently on leave, if could get input from you all on this voting process that would be much appreciated.

**Reply by loewenstein** [reply]
We had this for stack maintainers as well, didn't we and steering members just added their votes in the absence of enough maintainer, right? Sounds good for me again.

**Comment by mhdawson** [comment]
I think this can be closed now as I'm a stack maintainer.
